### PR TITLE
Replace `props.intl.formatMessage` with `<FormattedMessage />`

### PR DESCRIPTION
**Pull request by tony-ho**
Replace in-JSX occurrences of the `props.intl.formatMessage` expression with the `<FormattedMessage />` component.

The following conventions have been used for consistency with the existing code:
- Messages defined in `globalStrings` are passed using spread operator.
- Local messages props are passed inline to the `FormattedMessage` component and the key removed from the `messages` object.

Generated react-intl localizations have been updated.

Fixes meedan/check#20

### CODE REVIEW

File: src/app/components/team/TeamInviteMembers.js [hunk 1]
```diff
@@ -331,7 +327,7 @@ class TeamInviteMembers extends Component {
           scroll="paper"
           fullWidth
         >
-          <DialogTitle>{this.props.intl.formatMessage(messages.inviteMembers)}</DialogTitle>
+          <DialogTitle><FormattedMessage id="teamInviteMembers.newInvite" defaultMessage="Invite members" /></DialogTitle>
```
Comment (amoedoamorim) [inline src/app/components/team/TeamInviteMembers.js:330]: Please keep same `id`: `TeamInviteMembers.newInvite`

File: src/app/components/user/AttributionDialog.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { injectIntl } from 'react-intl';
+import { FormattedMessage, injectIntl } from 'react-intl';
```
Comment (amoedoamorim) [inline src/app/components/user/AttributionDialog.js:3]: Please remove `injectIntl` from the import and the respective usage

File: src/app/components/user/UserEmail.js [hunk 3]
```diff
@@ -114,10 +106,10 @@ class UserEmail extends React.Component {
           </CardContent>
           <CardActions>
             <Button onClick={this.handleClickSkip}>
-              {this.props.intl.formatMessage(messages.skip)}
+              <FormattedMessage id="uuserEmail.skip" defaultMessage="Skip" />
```
Comment (amoedoamorim) [inline src/app/components/user/UserEmail.js:109]: Please keep same `id`: `userEmail.skip`

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Hello @tony-ho ! Thank you so much for your contribution! We appreciate it! Sorry to take so long to review it! Please fix some minor details so we can accept and merge it! Also please review these files and remove `injectIntl` from the import statement where it's no longer needed.

**Comment by tony-ho** [comment]
@amoedoamorim Thanks for the review. The requested changes have been resolved and `injectIntl` has been removed where no longer used.

**Comment by danielafeitosa** [comment]
Hi @tony-ho !

For your contribution to CheckWeb during [Meedan Hacktoberfest 2020](https://meedan.com/hacktoberfest) you are eligible to earn a swag :tada: 
To receive it please fill in the form with your information: https://forms.gle/qKMRkWK7CEccCtGv5

Thank you!

**Pull request merged**